addon_service/configured_addon/link/models.py

```diff
@@ -20,34 +22,43 @@ def is_supported_resource_type(resource_type: int):
 
 
 class ConfiguredLinkAddon(ConfiguredAddon):
-    target_id = models.CharField()
-    int_resource_type = models.BigIntegerField(validators=[is_supported_resource_type])
+    target_id = models.CharField(
+        default=None, null=True
```

Comment:
How about `null=True, blank=True`?

Reply:
Works for me

Reply:
Update: I will change all to `blank=True` so we can do loose `Fasly` check instead of strict `None` Check